frontend/src/app/projects/project-detail/project-details.component.html

```diff
@@ -7,7 +7,14 @@
   <div class="grow basis-1">
     <div class="flex flex-wrap items-stretch">
       <app-project-metadata class="flex-1"></app-project-metadata>
-      <app-create-readonly-session class="flex-1"></app-create-readonly-session>
+      <app-create-readonly-session
+        class="flex-1"
+        *ngIf="projectType === 'general'"
+      ></app-create-readonly-session>
+      <app-provision-workspace
+        class="flex-1"
+        *ngIf="projectType === 'training'"
+      ></app-provision-workspace>
```

Comment:
```suggestion
      <app-create-readonly-session
        class="flex-1"
        *ngIf="(projectService.project$ | async)?.type === 'general'"
      ></app-create-readonly-session>
      <app-provision-workspace
        class="flex-1"
        *ngIf="(projectService.project$ | async)?.type === 'training'"
      ></app-provision-workspace>
```
 We can directly use the project observable via the `async` pipe here which safes us from using a separate variable in the `.ts` file and also significantly reduces the size of the `.ts` file:

```ts
/*
 * SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
 * SPDX-License-Identifier: Apache-2.0
 */

import { Component } from '@angular/core';
import { UntilDestroy } from '@ngneat/until-destroy';

import { ProjectUserService } from 'src/app/projects/project-detail/project-users/service/project-user.service';
import { ProjectService } from '../service/project.service';

@UntilDestroy()
@Component({
  selector: 'app-project-details',
  templateUrl: './project-details.component.html',
})
export class ProjectDetailsComponent {
  constructor(
    public projectService: ProjectService,
    public projectUserService: ProjectUserService,
  ) {}
}
```

We can also remove the `.css` file here as, at least from what I can see, the `title` id and the `metadata-item` class are not used in the html anymore.